docs/how-tos/entanglement_forging/use-asymmetric-bitstrings.ipynb

```diff
@@ -71,23 +65,25 @@
    "outputs": [],
    "source": [
     "# Define a molecular system of interest - Methyl radical\n",
-    "molecule = Molecule(\n",
-    "    geometry=[\n",
-    "        [\"C\", [0.0, 0.0, 0.00]],\n",
-    "        [\"H\", [1.0790, 0.0, 0.00]],\n",
-    "        [\"H\", [-0.5395, -0.9344, 0.00]],\n",
-    "        [\"H\", [-0.5395, 0.9344, 0.00]],\n",
+    "molecule = MoleculeInfo(\n",
+    "    symbols=\"CHHH\",\n",
+    "    coords=[\n",
+    "        (0.0, 0.0, 0.0),\n",
+    "        (1.0790, 0.0, 0.0),\n",
+    "        (-0.5395, -0.9344, 0.0),\n",
+    "        (-0.5395, 0.9344, 0.0),\n",
     "    ],\n",
-    "    charge=0,\n",
     "    multiplicity=2,\n",
     ")\n",
     "\n",
-    "driver = PySCFDriver.from_molecule(molecule=molecule, basis=\"sto-3g\")\n",
+    "driver = PySCFDriver.from_molecule(molecule)\n",
+    "driver.run()\n",
+    "problem = driver.to_problem(basis=ElectronicBasis.MO)\n",
```

Comment:
```suggestion
    "problem = driver.run()\n",
```
Unless you specifically want to show how one can request a specific basis there is no need to do it like this (since MO is the default).